### ISSUE TITLE
Molecule 3.0 pre-release check list

### ISSUE THREAD

**Issue by decentral1se** [ansible/molecule#2067]
OK, let's have it, what do we need?

* [ ] Ansible verifier as the default
  * What do we need to ensure here?
* [ ] Any critical / blocker bugs?
  * ~~https://github.com/ansible/molecule/issues/2070~~
  * https://github.com/ansible/molecule/issues/2069
* [ ] Change log complete
* [ ] ???

**Comment by decentral1se** [comment]
RE: ansible verifier, from a quick check, generating a new role still does have the testinfra configuration in place and the pytest files there. We should change that around. Perhaps a new default ansible verifier file with the same host check as testinfra?

**Comment by znerol** [comment]
I'd really love to see a 2.x (pre-)release before/now which is compatible with ansible 2.8. Is there any way I can help to make this happen?

**Comment by decentral1se** [comment]
@znerol, this is a Good Idea. I'm working on closing those critical bugs listed above and then I will try to get this next 2.X release out with Ansible 2.8 support. This will be good actually to get more feedback. Ansible will not be the default verifier but available for use.

**Comment by MarkusTeufelberger** [comment]
I don't see something so far that would warrant increasing the major release number.

**Comment by decentral1se** [comment]
> I don't see something so far that would warrant increasing the major release number.

New default verifier is Ansible.

**Comment by MarkusTeufelberger** [comment]
That's a minor detail in a template that gets rendered, no API change.

**Comment by decentral1se** [comment]
Molecule is not a regular library. The "API" is that you get testinfra as the default verifier on 2.X and on 3.X you have new a default verifier. On newly generated templates, you'll have to switch the configuration to use testinfra if that is what you want. That is a "breaking API change". This is following a number of people +1'ing the idea of 3.0 release on a recent WG meeting.

**Comment by MarkusTeufelberger** [comment]
Molecule 1 --> 2 was a major rewrite. Molecule 2 --> 3 happens because a default value in `molecule init role` changes? Also I'd like to see at least 2-3 releases with the Ansible verifier as non-default option to find out if it might even make sense to keep around. Ansible is not a verification tool.

**Comment by decentral1se** [comment]
I've got better things to be doing.

**Comment by seandst** [comment]
ansible is currently not the default verifier; the PR to make ansible the default verifier (#2013) was closed without merging. I also just tested `molecule init role` on latest master and it's still using testinfra as expected.

If changing the default verifier was the main reason for considering a major version bump, should this issue be reopened and renamed to reference molecule 2.21, or should a new issue be created?
